README.md

```diff
@@ -338,51 +338,86 @@ Once complete your base AWS resources should be in place
 
 * [kubectl](https://kubernetes.io/docs/user-guide/prereqs/)
 * [Kops](https://github.com/kubernetes/kops)
-* jq
-* yq
+* Python 3.6+
 
-(On macOS you can: `brew install kubectl kops jq yq`)
+(On macOS you can: `brew install kubectl kops python3`)
 
-2. Copy an existing cluster config:
-```
-cp -R infra/kops/clusters/alpha infra/kops/clusters/$ENVNAME
-```
+2. Create a values YAML file for Kops:
+  Most information required by Kops can be obtained from Terraform, with the exception of 
+  authentication information, cluster size, and instance machine image. This information 
+  must be supplied as a YAML file with the following structure, as 
+  `infra/kops/config/$ENVNAME.yaml`:
 
-3. Set the correct values for your new cluster config:
-```
-cd infra/terraform/global
-export KOPS_STATE_STORE=s3://`terraform output kops_bucket_name`
-export ENV_DOMAIN=`terraform output -module=cluster_dns dns_zone_domain`
-# Get the Client ID from Auth0 "kubectl-oidc" application e.g.
-export KUBECTL_OIDC_CLIENT_ID=P742wMtS4iiA6axtbPd2ygpOa64gZqGD
+  ```
+  OIDC:
+    ClientID: ""
+
+  instanceGroups:
+    masters:
+      machineType: ""
+
+    nodes:
+      machineType: ""
+      rootVolumeSize: ""
+      maxSize: ""
+      minSize: ""
+
+    bastions:
+      machineType: ""
+      maxSize: ""
+      minSize: ""
+  ```
 
-cd ../../../infra/terraform/platform
-../../kops/configure.sh $KOPS_STATE_STORE $ENVNAME $ENV_DOMAIN $KUBECTL_OIDC_CLIENT_ID
-```
+  | Key                    | Value                                                    |
+  |------------------------|----------------------------------------------------------|
+  | `OIDC.ClientID`        | OIDC client ID from the `kubectl-oidc` Auth0 application |
```

Comment:
Just flagging that kubectl-oidc is not created in Auth0 until later on in these instructions. Consider bringing those forward in the order. (Obvs these are set to be replaced by terraform goodness, but can't hurt)